ted/src/main/java/frc/robot/subsystems/LEDSubsystem.java

```diff
@@ -0,0 +1,166 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+// ************************************************************
+// Bishop Blanchet Robotics
+// Home of the Cybears
+// FRC - Crescendo - 2024
+// File: LEDSubsystem.java
+// Intent: Forms a subsystem to control the LEDs
+// ************************************************************
+
+
+// ʕ •ᴥ•ʔ ʕ•ᴥ•  ʔ ʕ  •ᴥ•ʔ ʕ •`ᴥ´•ʔ ʕ° •° ʔ ʕ •ᴥ•ʔ ʕ•ᴥ•  ʔ ʕ  •ᴥ•ʔ ʕ •`ᴥ´•ʔ ʕ° •° ʔ
+
+package frc.robot.subsystems;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.BooleanSupplier;
+import java.util.Iterator;
+
+import frc.robot.common.LEDStateAction;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.common.LEDState;
+import com.ctre.phoenix.led.*;
+import com.ctre.phoenix.led.CANdle.LEDStripType;
+import com.ctre.phoenix.led.CANdle.VBatOutputMode;
+
+public class LEDSubsystem extends SubsystemBase {
+    private final CANdle leds;
```

Comment:
there seem to be indentation differences in this file. Can you use VS code to format the file properly?

Reply:
done